integration/setup/setup.go

```diff
@@ -192,3 +200,38 @@ func installAWSResource() error {
 
 	return nil
 }
+
+func installCredential(h *framework.Host) error {
+	o := func() error {
+		k8sClient := h.K8sClient()
+
+		k8sClient.CoreV1().Secrets("giantswarm").Delete(credentialName, &metav1.DeleteOptions{})
```

Comment:
There credenitalName constant, but literal is used for the namespace, I think it would be good if both were either consts or literals.

Reply:
done